src/parse/parser.rs

```diff
@@ -964,9 +957,7 @@ impl<'help, 'app> Parser<'help, 'app> {
             parser.app.g_settings = self.app.g_settings;
         }
 
-        if parser.app.bin_name != self.app.bin_name {
-            parser.app.bin_name = Some(format!("{} {}", bin_name, parser.app.name));
-        }
```

Comment:
Any idea what the intention of this removed check was?

Oh, is this adding that last `sc.name` that we skipped earlier?

Reply:
Yup

Reply:
Yup.